sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/IngestionResponseHelper.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals.Diagnostics;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    internal static class IngestionResponseHelper
+    {
+        public static string?[] GetErrorsFromResponse(Response? response)
```

Comment:
Could you please validate the logic from the below code and confirm if its on the same lines. https://github.com/microsoft/ApplicationInsights-dotnet/blob/248800626c1c31a2b4100f64a884257833b8c77f/BASE/src/ServerTelemetryChannel/Implementation/TransmissionPolicy/ErrorHandlingTransmissionPolicy.cs#L40C89-L40C89

Reply:
It's very similar, almost identical datamodels but using a different deserializer.
AI SDK only does the deserialization when logging is Verbose. I think this is a good idea and would mitigate our perf concerns.